src/main/java/com/shippingcostestimator/enterprise/dto/Destination.java

```diff
@@ -12,6 +12,6 @@
  */
 public @Data
 class Destination {
-    String startAddress;
-    String endAddress;
+    private String startAddress;
+    private String endAddress;
```

Comment:
Added 'private' to the attributes so it'll follow the encapsulation concept for Oobject Oriented Programming. [Ref](https://www.geeksforgeeks.org/encapsulation-in-java/)

Reply:
Good.